programs/server/config.xml

```diff
@@ -134,6 +134,34 @@
     <max_connections>4096</max_connections>
     <keep_alive_timeout>3</keep_alive_timeout>
 
+    <!-- gRPC protocol (see src/Server/grpc_protos/clickhouse_grpc.proto for the API)
+    <grpc_port>9001</grpc_port>
+    <grpc>
+        <enable_ssl>true</enable_ssl> -->
```

Comment:
This --> is strange.

Reply:
xml doesn't support nested comments.

Reply:
Yes, but it looks like you close this comment too early.

Reply:
What do you think about [this style](https://stackoverflow.com/questions/1324821/nested-comments-in-xml/35479062#35479062)?

Reply:
It looks surprising to the reader, I'm sure almost no one will understand it from the first glance.